<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: rgb(255, 255, 255);
	background-color: rgb(27, 27, 27);
	font: 12pt &quot;Trender&quot;;
}


QPushButton {
color: 'white';
background-color: rgb(46, 46, 46);
border-style: outset;
border-width: 2px;
border-color: rgb(64, 0, 0);
padding: '20px';
border-radius: '10px'; 
font: 12pt &quot;Trender&quot;;
}

QPushButton:pressed {
background-color: rgb(27, 27, 27)
}


QSlider::groove:horizontal {
    border: 1px solid #aa0000;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: rgb(35, 35, 35);
    margin: 10px 10;
	border-radius: 4px;
}

QSlider::handle:horizontal {
	border: 4px solid rgb(255, 0, 0);
    background: rgb(35, 35, 35);
    width: 16px;
    margin: -8px -8; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 12px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="qwidget_left" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="qlabel_chairbot_logo">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/stuff/chairbot.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="qwidget_sliders" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#qwidget_sliders {
	border-style: outset;
	border-width: 2px;
	border-color:rgb(64, 0, 0);
}
QWidget {
	background-color: rgb(35, 35, 35);
}
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="qvlayout_slider_labels">
            <item>
             <widget class="QLabel" name="qlabel_translation_limit">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);
padding: 10px</string>
              </property>
              <property name="text">
               <string>translation limit</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="qlabel_twist_limit">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);
padding: 10px</string>
              </property>
              <property name="text">
               <string>twist limit</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="qvlayout_sliders">
            <item>
             <widget class="QSlider" name="qslider_translation">
              <property name="styleSheet">
               <string notr="true">QSlider {
	padding: 9px;
	background-color: rgb(50, 50, 50);
}</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="qslider_twist">
              <property name="styleSheet">
               <string notr="true">QSlider {
	padding: 9px;
	background-color: rgb(50, 50, 50);
}</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="qvlayout_percent_labels">
            <item>
             <widget class="QLabel" name="qlabel_translation_percent">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);
padding: 10px</string>
              </property>
              <property name="text">
               <string>30%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="qlabel_twist_percent">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);
padding: 10px</string>
              </property>
              <property name="text">
               <string>50%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="qwidget_button_holder" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#qwidget_button_holder {
	border-style: outset;
	border-width: 2px;
	border-color:rgb(64, 0, 0);
	background-color: rgb(35, 35, 35);
}

</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QPushButton" name="qbutton_control_mode_switch">
         <property name="styleSheet">
          <string notr="true">margin-top: 50px;
margin-bottom: 50px;</string>
         </property>
         <property name="text">
          <string>switch to onboard control</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="qlayout_connections">
         <property name="topMargin">
          <number>11</number>
         </property>
         <item>
          <widget class="QLabel" name="qlabel_nt_connected">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>nt disconnected</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qbutton_swap_sim">
           <property name="text">
            <string>swap sim/robot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
